python/cuvs/cuvs/test/test_distance.py

```diff
@@ -79,7 +80,5 @@ def test_distance(n_rows, n_cols, inplace, order, metric, dtype):
     actual = output_device.copy_to_host()
 
     tol = 1e-3
-    if np.issubdtype(dtype, np.float16):
-        tol = 1e-1
```

Comment:
I think I added this reduced tolerance because I was seeing failures - is this no longer needed?

Reply:
Yeah, I tried it successfully at local machines. I think this change can help us block potential actual failures in the future, so I made it.